app/models/health/health_file.rb

```diff
@@ -28,19 +28,21 @@ def file_not_too_large
     end
 
     def valid_file_type
-      errors.add :file, "File must be a PDF" if content_type != 'application/pdf'
+      errors.add :file, 'File must be a PDF' if content_type != 'application/pdf'
     end
 
     def title
       self.class.model_name.human
     end
 
     def signature
-      try(:participation_form).try(:signature_on) || try(:release_form).try(:signature_on)
+      try(:release_form).try(:participation_signature_on) || try(:release_form).try(:signature_on)
```

Comment:
this is checking any signature, not fully signed.  Is that correct?

Reply:
This is a strange bit of STI behavior, refactored.